config/config.go

```diff
@@ -40,7 +42,15 @@ func Init(configFilePath string) error {
 
 	// load config
 	if err := K.Load(file.Provider(configFilePath), yaml.Parser()); err != nil {
-		return fmt.Errorf("load: %w", err)
+		return fmt.Errorf("load file: %w", err)
+	}
+
+	// load environment variables
+	if err := K.Load(env.Provider("TQM_", ".", func(s string) string {
+		return strings.Replace(strings.ToLower(
+			strings.TrimPrefix(s, "TQM_")), "_", ".", -1)
```

Comment:
Should we do double underscore for for the prefix `TQM__` for whatever issues the kubernetes users had with autobrr env vars?

Reply:
Changed now!